aws/cloudformation/standalone/gen_ai_curriculum/config.rb

```diff
@@ -0,0 +1,66 @@
+require "ostruct"
+require_relative '../../../../lib/cdo/shared_constants'
+
+module Config
+  # Configuration for each endpoint used in the Gen AI Curriculum. Required properties:
+  #   hf_model_id: HuggingFace model ID
+  #   model_id: App-wide model ID (from shared constants)
+  #   model_name: Short name for naming resources (letters only)
+  #   instance_type: EC2 Instance Type for endpoint
+  #   min_num_instances (prod only): Minimum number of deployed instances used for autoscaling on production
+  #     on test, only a single instance will be used without autoscaling to limit resource usage
+  #   max_num_instances (prod only): Maximum number of deployed instances used for autoscaling on production
+  #     not used on test
+  #   autoscaling_target_value (prod only): Number of requests / minute at which autoscaling starts to take effect
+  #     not used on test
+  ENDPOINT_CONFIGS = [
```

Comment:
Note that this is currently missing the BioMistral endpoint since I was having some trouble deploying it specifically. Will keep troubleshooting that in parallel.

Reply:
Was able to get BioMistral working with a longer timeout and larger instance (thanks @snickell for the tip!)